### PR TITLE
feat: new project media content

### PR DESCRIPTION
**Pull request by wgwz**
## Description

Closes: https://github.com/regen-network/regen-registry/issues/1631

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] provided a link to the relevant issue or specification
- [ ] reviewed "Files changed" and left comments if necessary

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**.*

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] reviewed documentation is accurate
- [ ] manually tested (if applicable)

### CODE REVIEW

File: shacl/projects/common.ttl [hunk 1]
```diff
@@ -90,8 +90,27 @@ regen:ProjectTypeOptionalPropertyShape sh:path regen:projectType ;
 .
 
 regen:PreviewPhotoPropertyShape sh:path regen:previewPhoto ;
-  sh:datatype schema:URL ;
   sh:group regen:ProjectPageMediaGroup ;
+  sh:or (
+    [
+      sh:datatype schema:URL ;
+    ]
```
Comment (blushi) [inline shacl/projects/common.ttl:97]: I believe we should get rid of this part completely 
even when there's no credit text, the preview photo field structure should still be the same:
```json
"regen:previewPhoto": {
   "schema:url" : "https://some.url"
}
```

else that would just complicates things unnecessarily on the front-end

File: shacl/projects/project-page.ttl [hunk 2]
```diff
@@ -21,9 +21,13 @@ regen:ProjectPageShape a sh:NodeShape ;
...
   sh:property [
-    sh:path schema:creditText ;
-    sh:datatype xsd:string ;
+    sh:path regen:storyMedia ;
+    sh:minCount 1 ;
     sh:maxCount 1 ;
+    sh:or (
+      [ sh:class regen:Photo ]
+      [ sh:class regen:Video ]
+    )
```
Comment (blushi) [inline shacl/projects/project-page.ttl:30]: I like this approach! but what about using existing types from schema.org instead?
https://schema.org/VideoObject
https://schema.org/ImageObject

Comment (wgwz) [reply shacl/projects/project-page.ttl:30]: yeah, i like that! looks like that should be compatible

File: jsonld/projects/project-page.jsonld [hunk 3]
```diff
@@ -2,27 +2,30 @@
...
   "schema:description": "",
-  "regen:previewPhoto": "",
+  "regen:previewPhoto": {
+    "schema:url": "",
+    "schema:creditText": ""
+  },
   "regen:galleryPhotos": [
-    ""
+    {
+      "schema:url": ""
```
Comment (blushi) [inline jsonld/projects/project-page.jsonld:22]: let's also add optional credit and caption field here?

**Review by blushi — APPROVED** [review]

**Pull request merged**